JS-week-2/weather_wear.js

```diff
@@ -0,0 +1,10 @@
+function getWhattoWear (temInCelcius){
```

Comment:
Just a small naming suggestion: when using [`camelCase` naming](https://developer.mozilla.org/en-US/docs/Glossary/Camel_case) the first letter of every word (except for the first one) should be capitalised. 

It's just a naming convention we (normally) use to name variables, functions etc (but of course, this can be project/team/company-specific). There are many other naming conventions, for example in Python or Ruby it is recommended to use `underscore_case`.

So, in your case this could be named:
```suggestion
function getWhatToWear (temInCelcius){
```

Reply:
P.S. The function looks and works great!

Reply:
Done. Sorry for the late.